lib/api/asset/v1/types.ts

```diff
@@ -61,6 +61,13 @@ export interface AssetManagement {
   managingOrganisationId: string;
   owner: string;
   isTMOManaged: boolean;
+  propertyOccupiedStatus?: string;
+  propertyOccupiedStatusReason?: string;
+  isNoRepairsMaintenance?: boolean;
+  councilTaxType?: string;
+  councilTaxLiability?: string;
+  isTemporaryAccomodation?: boolean;
+  readyToLetDate?: boolean;
 }
 
```

Comment:
For reference:
![image](https://github.com/LBHackney-IT/mtfh-frontend-common/assets/70756861/ccecdb3b-2c84-4711-b1f3-d793af1592e6)